packages/web/src/routes/secrets.tsx

```diff
@@ -264,13 +282,23 @@ function SecretRow(props: {
 function NewSecretForm(props: {
   setError: (message: string | null, clearAfter?: number | null) => void;
 }) {
+  useHotkeys(
```

Comment:
In all cases I've seen thus far for inputs, i find this pattern to be less obvious (the code is located somewhere other than the `Input`) and heavier weight than:

```jsx
onKeydown={(e) => e.key === 'Enter' && onSubmit()}
```